bot/exts/fun/wolfram.py

```diff
@@ -0,0 +1,75 @@
+from random import choice
```

Comment:
It would be more understandable if we do `random.choice()` rather than `choice()`.
```suggestion
import random
```